main.c

```diff
@@ -682,6 +684,8 @@ void on_buttonpress(XButtonEvent *bev)
 	prefix = 0;
 }
 
+const struct timespec ten_ms = {0, 10000000};
```

Comment:
ten_ms should not be global, see: https://github.com/nsxiv/nsxiv/commit/0c66c0e25f8d3aa90a1d4e7bfaed734e30a29e4c